composer/loggers/mosaicml_logger.py

```diff
@@ -150,6 +150,10 @@ def close(self, state: State, logger: Logger) -> None:
         if self._enabled:
             wait(self._futures)  # Ignore raised errors on close
 
+    def log_checkpoint_upload_start(self, state: State) -> None:
+        self._log_metadata({'checkpoint_upload_start_time': [time.time(), state.timestamp.batch.to_timestring()]})
```

Comment:
looks fine, would definitely double check if json is created from iterable properly 

https://github.com/mosaicml/composer/blob/4e1eb8d20a904510c2c03819f4517c1f12fe7b3a/composer/loggers/mosaicml_logger.py#L262

Reply:
i guess theres a test case already 

https://github.com/mosaicml/composer/blob/4e1eb8d20a904510c2c03819f4517c1f12fe7b3a/tests/loggers/test_mosaicml_logger.py#L71 

Good that this change is so simple :) Dont think we use the checkpoint load event anywhere yet in mcloud, so ok to change to list. But @jjanezhang  should confirm here